dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.microprofile.reactive.streams.operators-1.0.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.microprofile.reactive.streams.operators-1.0
```

Comment:
As discussed, I think we should leave out "operators" from the name?

Reply:
Understood - I will check with Emily.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/4492